src/scripts/main.js

```diff
@@ -1 +1,84 @@
 'use strict';
+
+const button = document.querySelector('body');
```

Comment:
The `button` variable is currently selecting the entire `body` element. If the intention is to select a specific button, you should use a more specific selector, such as `document.querySelector('button')` or an appropriate class or ID selector.